webcompat/api/uploads.py

```diff
@@ -100,6 +106,10 @@ def save(self):
             save_parameters['save_all'] = True
         # unpacking save_parameters
         self.image_object.save(file_dest, **save_parameters)
+        # Creating the thumbnail
+        size = (150, 150)
+        self.image_object.thumbnail(size, Image.ANTIALIAS)
```

Comment:
@miketaylr sure. Here we go.
![original](https://cloud.githubusercontent.com/assets/505230/19329145/36e10ff2-9112-11e6-8ecb-d040d433d241.png)